types/null/sf_point.go

```diff
@@ -130,28 +127,21 @@ func (p SFPoint) Value() (driver.Value, error) {
 }
 
 // Scan implements the database/sql Scanner interface. It expects to receive a
-// valid WKB encoded byte describing a Point as a []byte, or NULL as a nil from
-// an SQL database. A zero-length []byte or a nil will be considered NULL, and p
-// will be nulled. Otherwise, the value will be passed to types.SFPoint to be
-// scanned and parsed as a WKB Point.
+// valid WKB encoded []byte describing a Point, or NULL as a nil from
+// an SQL database. A nil will be considered NULL, and p will be nulled.
+// Otherwise, the value will be passed to types.SFPoint to be scanned and parsed
+// as a WKB Point.
 func (p *SFPoint) Scan(src interface{}) error {
 	if p == nil {
 		return fmt.Errorf("null.SFPoint: Scan called on nil pointer")
 	}
 	switch x := src.(type) {
 	case nil:
-		p.Point = types.SFPoint{}
 		p.Valid = false
 		return nil
 	case []byte:
-		if len(x) == 0 {
```

Comment:
Some cursory research indicates that it doesn't actually store empty geometry values, but just in case it can in some situations, I'll add this back in.